src/client/pages/Offer/Introduction/HeroOfferDetails.tsx

```diff
@@ -103,23 +104,45 @@ const EditDetailsButton = styled(Button)`
   }
 `
 
+const getAddress = (quotes: BundledQuote[]) => {
+  const hasAddress = (quote: BundledQuote) => {
+    const interestedInFields = ['street', 'zipCode'] as const
+    return interestedInFields.every((field) => quote.data[field] != null)
+  }
+
+  const parseAddress = (address: Address) => {
+    const { street, apartment, floor } = address
+
+    return street
+      .concat(floor ? `, ${floor}.` : '')
+      .concat(apartment ? ` ${apartment}` : '')
+  }
```

Comment:
No, I can move it outside.

🤔 It's pretty much the same implementation we're using before with `InsuranceSummaryDetails` [getAddress](https://github.com/HedvigInsurance/web-onboarding/pull/750/files#diff-7c371857b44f7fecfd1b97f2c85ceea91357515d3c034c39faaba60335e4156eL13) function. But I understand your point as this could be an opportunity to reevaluate it. Gonna give a look.

Reply:
Cool, but it that's the way it worked before we can keep it :)

Reply:
Yeah, I've checked but it feels any solution that takes locale into account would be premature right now so I'm just leave as it is :)